frontend/src/component/admin/auth/ScimSettings/ScimSettings.tsx

```diff
@@ -138,6 +173,57 @@ export const ScimSettings = () => {
                         />
                     </Grid>
                 </Grid>
+
+                <Grid container spacing={3}>
+                    <Grid item md={10.5} mb={2}>
+                        <StyledTitleDiv>
+                            <strong>Delete SCIM Users</strong>
+                        </StyledTitleDiv>
+                        <p>
+                            This will remove all SCIM users from the Unleash
+                            database. This action cannot be undone through
+                            Unleash but the upstream SCIM provider may re sync
+                            these users.
+                        </p>
+                    </Grid>
+                    <Grid item md={1.5}>
+                        <Button
+                            variant='outlined'
+                            color='error'
+                            disabled={loading}
+                            onClick={() => {
+                                setDeleteUsersDialog(true);
+                            }}
+                        >
+                            Delete Users
+                        </Button>
+                    </Grid>
+                    <Grid item md={10.5} mb={2}>
+                        <StyledTitleDiv>
+                            <strong>Clear SCIM Groups</strong>
```

Comment:
```suggestion
                            <strong>Delete SCIM Groups</strong>
```
Maybe for consistency with the previous title about SCIM users?